mbuild/formats/gomc_conf_writer.py

```diff
@@ -0,0 +1,4467 @@
+import datetime
+import os
+import mbuild.formats.charmm_writer as mf_charmm
+from warnings import warn
+
+
+def dict_keys_to_list(dict):
+    """
+    Converts the dictionary keys into a list
+
+    Parameters
+    ----------
+    dict : dict
+        A provided dictionary
+
+    Returns
+    ---------
+    list : list
+        list of keys from the provided dictionary
+    """
+    list = [key for key in dict.keys()]
```

Comment:
Dont even need a temporary variable name before we use the return statement.

```suggestion
    return [key for key in dict.keys()]
```

Is all that is needed.

Reply:
Make sure to remove the following `return` statement if  you do the suggestion above.

Reply:
changed as requested.